doc/tutorials/python/01-lennard_jones/01-lennard_jones.tex

```diff
@@ -129,7 +129,7 @@ \section{Introduction}
 
 \section{Background}
 Today's research on Soft Condensed Matter has brought the needs for having a 
-flexible, extensible, reliable and efficient (parallel) molecular simulation 
+flexible, extensible, reliable, and efficient (parallel) molecular simulation 
```

Comment:
This is a `serial comma', see: [https://www.oxforddictionaries.com/words/comma#commas_in_lists]